flow/connectors/postgres/client.go

```diff
@@ -197,6 +197,44 @@ func (c *PostgresConnector) checkSlotAndPublication(slot string, publication str
 	}, nil
 }
 
+// GetSlotInfo gets the information about the replication slot size and LSNs
+func (c *PostgresConnector) GetSlotInfo(slotName string) ([]*protos.SlotInfo, error) {
```

Comment:
why does this return an array if it's matching on only one name

Reply:
if slotName is empty its returns info for all slots. Add a comment Amogh.